lib/hashes.md

```diff
@@ -5,40 +5,40 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A Hash is a collection of data elements that are indexed by a key that is another data element.
```

Comment:
I don't think there is anything incorrect about your answer, but I'd like to encourage you to be a little more precise. 

For some hash:
`hash = {"key1" => "value1", "key2" => "value2", "key3" => "value3"}`

The hash is a _collection_ of _key-value pairs_, indexed on the _keys_. Both the _key_ and _value_ can be any kind of Ruby Object.